src/main.rs

```diff
@@ -173,3 +220,15 @@ fn main() -> Result<(), Box<dyn Error>> {
 
     process::exit(0);
 }
+
+fn get_default_config_path() -> String {
```

Comment:
Looking at https://github.com/whitequark/rust-xdg/blob/master/src/lib.rs I thought that this might not handle windows, but following through to https://crates.io/crates/dirs, it looks like it is cross-platform after all.

Reply:
Nice! Rust libraries have been pretty good at Windows compat in my experience.

Reply:
Oh, actually. Nope. >_< Missed the first line:

```rust
#![cfg(any(unix, target_os = "redox"))]
```

This doesn't work on Windows at all.

Reply:
We could do something like this: https://github.com/DanielKeep/cargo-script/blob/614e60e5932e218ebff1e471303eb0d59870d03b/src/platform.rs#L309-L363

But for now I'm defaulting to `none` on Windows.